Crash when displaying a mesh or viewing an attachment

I get a crash whenever I try to view an attachment or object.

Radegast 2.48.456.0

The last log items are:

```
13:28:00 [ERROR] - <Fluppy288 Magic>: Async Packet Event Handler: System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'AttachmentDetail'.
   at System.Windows.Forms.Control.MarshaledInvoke(Control caller, Delegate method, Object[] args, Boolean synchronous)
   at System.Windows.Forms.Control.Invoke(Delegate method, Object[] args)
   at Radegast.AttachmentDetail.UpdateControls()
   at Radegast.AttachmentDetail.Objects_ObjectProperties(Object sender, ObjectPropertiesEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenMetaverse.ObjectManager.ObjectPropertiesHandler(Object sender, PacketReceivedEventArgs e)
   at OpenMetaverse.PacketEventDictionary.ThreadPoolDelegate(Object state)
13:28:05 [INFO] - Initializing OpenGL mode: Index: , Color: 32 (8888), Depth: 24, Stencil: 8, Samples: 0, Accum: 0 (0000), Buffers: 2, Stereo: False
```

--- a/Radegast/GUI/Consoles/AttachmentDetail.cs
+++ b/Radegast/GUI/Consoles/AttachmentDetail.cs
@@ -77,6 +77,11 @@ namespace Radegast
 
         private void UpdateControls()
         {
+            if (Disposing || IsDisposed)
+            {
+                return;
+            }
+
             if (InvokeRequired)
             {
                 Invoke(new MethodInvoker(UpdateControls));

--- a/Radegast/GUI/Rendering/PrimWorkshop.cs
+++ b/Radegast/GUI/Rendering/PrimWorkshop.cs
@@ -552,6 +552,11 @@ namespace Radegast.Rendering
                         SafeInvalidate();
                     });
 
+                    if (Disposing || IsDisposed)
+                    {
+                        break;
+                    }
+
                     if (!instance.MonoRuntime || IsHandleCreated)
                     {
                         BeginInvoke(loadOnMainThread);

